climada/hazard/relative_cropyield.py

```diff
@@ -201,13 +203,16 @@ def set_from_single_run(self, input_dir=None, bbox=BBOX,
     def calc_mean(self, yearrange=(YEARCHUNKS[SCENARIO[0]])['yearrange'], save=False,
                   output_dir=OUTPUT_DIR):
         """Calculates mean of the hazard for a given reference time period
-            Parameters:
+
+            Optional Parameters:
                 yearrange (array): time period used to calculate the mean intensity
                 default: 1976-2005 (historical)
+            save (boolean): save mean to file? default: False
+            output_dir (str): path of output directory
 
             Returns:
                 hist_mean(array): contains mean value over the given reference
-                time period for each centroid
+                    time period for each centroid
         """
         time_array = np.array(self.event_name, dtype=int)
         idx_time = np.zeros(2, dtype=int)
```

Comment:
The idx_time is not really needed. This part can be written in a simpler/more straightforward way.

Also, it is not ideal to have hardcoded indices on the yearrange variable. If in the future another yearrange is added, the code would be broken. Can this be written in a more flexible way?